pipelines/batch/src/main/java/com/google/fhir/analytics/JdbcResourceWriter.java

```diff
@@ -139,12 +161,25 @@ static void createTables(FhirEtlOptions options)
                         .collect(Collectors.toList())));
             builder.append(");");
             createSingleTable(jdbcSource, builder.toString());
+            // We create an index on the id column to speed up the incremental update process. It is
+            // also useful for common JOIN scenarios where resource IDs are involved.
+            createIdIndex(jdbcSource, vDef.getName());
           }
         }
       }
     }
   }
 
+  private static void deleteOldViewRows(DataSource dataSource, String tableName, String resId)
```

Comment:
Can we have this API exposed to the callers, so that they can delete the rows based on the identifier?